project/Build.scala

```diff
@@ -598,7 +625,8 @@ object Build extends sbt.Build {
 
   lazy val delambdafySetting = {
     scalacOptions ++= (
-        if (scalaBinaryVersion.value == "2.10") Seq()
+        if (isGeneratingEclipse) Seq()
+        else if (scalaBinaryVersion.value == "2.10") Seq()
```

Comment:
I use if/else if when using `||` makes the line too long ^^
But I should probably use if/else if everywhere, indeed.